tests/check/check_h26xsigned_sign.c

```diff
@@ -581,53 +581,69 @@ START_TEST(undefined_nalu_in_sequence)
 }
 END_TEST
 
-START_TEST(two_seis_in_buffer)
+/* Test description
+ * Verify that after 2 completed Sei's created ,they will be transfered in
+ * correct order
+ * The operation is as follows:
+ * 1. Setup a signed_video_t session
+ * 2. Add  2 I NALU for signing that will trigger 2 Sei Nalu
+ * 3. Get the Sei NALU's to prepend
+ * 4. Check that Sei NALU's transfered in  correct order
+ */
+START_TEST(two_completed_seis_pending)
```

Comment:
You should have the same test but when using `signed_video_get_sei(...)`
Name this test `two_completed_seis_pending_legacy` and keep the name with the new API

Reply:
Currently signed_video_get_sei(..) will  get the first element in SEI buffer  .Should I modify  it such a way that if there is more then 1 completed SEI waiting in the buffer it will behave like  signed_video_get_nalu_to_prepend(..)